integration-tests/src/test/java/oracle/kubernetes/operator/ITSitConfig.java

```diff
@@ -41,17 +49,27 @@
   @BeforeClass
   public static void staticPrepare() throws Exception {
     // initialize test properties and create the directories
-
     if (!QUICKTEST) {
       // initialize test properties and create the directories
       initialize(APP_PROPS_FILE);
-
       if (operator1 == null) {
         operator1 = TestUtils.createOperator(OPERATOR1_YAML);
       }
       TESTSCRIPTDIR = BaseTest.getProjectRoot() + "/integration-tests/src/test/resources/";
+      // Create the MySql db container
+      ExecResult result =
+          TestUtils.exec(
+              "kubectl create -f " + TESTSCRIPTDIR + "/sitconfig/mysql/mysql-dbservices.yml");
```

Comment:
please make sure to update the cleanup.sh to remove this database as well, in case test fails

Reply:
Good catch Mark.

The key to that is to make sure (a) the DB k8s resource are decorated with 'domain uid' labels so the cleanup script can find them - same as we do for secrets and the like, (b) the nfs files are in the same 'directory tree' as WL's nfs files, and (c) give the files the same uid/gid as the WL files.   I'll send you a slack on the latter if you need help with that.

If you don't do all three, then cleanup may work fine on some platforms, but not on others.   @vanajamukkara has the latest results in this area...

Reply:
We can do clean up the mySQL DB instance in AfterClass method explicitly using kubectl delete -f  mysql-dbservices.yml.  Here we are not saving any any data from mySQL after the test run.

Reply:
If mySQL isn't storing data then only (a) is needed.   

A 'delete' in unprepare is insufficient.   The test may crash partway through and never call unprepare, but we still need cleanup.sh to clean up *everything*.   Otherwise the next run may fail.

Reply:
Currently, as Pani said the kubectl delete -f mysql-dbservices.yml is in the AfterClass.
Discussed it with Vanaja to create another JIRA task to create/delete MySQL db to be used by any tests with mysql DB decorated with domain uid label. So that cleanup.sh can clean up the DB in case if the test crashes.
Can we have that as a separate PR ?

Reply:
I don't think we should check-in integration tests unless it's guaranteed cleanup.sh can  clean them up.

Reply:
Adding mysql pods with domainUID labels so that it can get cleaned up in case the test halts midway. Also added mysql in cleanup.sh